### PR TITLE
Fix call extends helpers in admin side

### PR DESCRIPTION
**Pull request by shandak**

### CODE REVIEW

File: mvcoverride.php [hunk 1]
```diff
@@ -226,7 +226,7 @@ private function getOverrideFileInfo($includePath, $component, $filePath, $type
 			$baseName = basename($filePath);
 			$prefix = substr($baseName, 0, 5);
 
-			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') )
+			if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin() && $prefix != 'admin') || !(!$app->isAdmin() && $prefix == 'admin'))
```
Comment (Kixo) [inline mvcoverride.php:229]: (!$app->isAdmin() && $prefix != 'admin') || !(!$app->isAdmin() && $prefix == 'admin')
is actually just (!$app->isAdmin())

Comment (shandak) [inline mvcoverride.php:229]: You missing one more ! before (!$app->isAdmin() && $prefix == 'admin')
Its different

Comment (Kixo) [inline mvcoverride.php:229]: yes but those two negate each other... so you could have it like this:
if (($app->isAdmin() && $prefix == 'admin') || (!$app->isAdmin()))
it is the same check as yours above

Comment (shandak) [inline mvcoverride.php:229]: Right rule, what i try add - if i create helper for using in admin side, but original file in frontend side(frontend helper files not have in name `admin` prefix) - you example return false.

**Pull request merged**